elastic/kibana
# [Infrastructure UI] Fix time range filter in alerts query

## 📓 Summary

Closes #153741 

This PR fixes the time range filter by using the `kibana.alert.time_range` field instead of `@timestamp`.

## 🧪 Testing

- Navigate to Hosts View
- Create an Inventory Alert that will trigger immediately
- Refresh the search until some alerts are triggered
- Play with relative time range (eg. 15 min ago -> 2 min ago) to verify alerts appears correctly
